eng/common/sdl/execute-all-sdl-tools.ps1

```diff
@@ -0,0 +1,97 @@
+Param(
```

Comment:
Why are these files part of the PR? thought these were already flowing with arcade?

Reply:
Oops, missed removing from PR. Merged master , this should be fixed now.